spec/feature_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'takeaway'
+
+describe 'User stories' do
+
+  # 1 User Story
+  #As a customer
+  #So that I can check if I want to order something
+  #I would like to see a list of dishes with prices
+  it 'see a list of dishes with prices ' do
+    takeaway = Takeaway.new
+    expect(takeaway.display_menu).to eq ('{"chicken"=>3, "rice"=>1, "pizza"=>6, "chips"=>2}')
```

Comment:
Lint/ParenthesesAsGroupedExpression: `(...)` interpreted as grouped expression. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)<br>Metrics/LineLength: Line is too long. [93/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/RedundantParentheses: Don't use parentheses around a literal.